src/runtime/api-channel.ts

```diff
@@ -166,6 +171,13 @@ class ThingMapper {
     return id;
   }
 
+  reCreateMappingForThing(thing) {
```

Comment:
nit: suggest lowercasing the 'C', since the first full verb in the method name is 'recreate'.

Reply:
Changed!